docs/authoring.md

```diff
@@ -78,40 +76,40 @@ To generate a NuGet package for the component, you can simply right click on the
+You can then use the Visual Studio UI to add a reference to the C#/WinRT component's `csproj` file, and you also need to add a reference to the WinMD file produced 
+for your component. The WinMD can be found in the output (`bin`) directory and the `Generated Files` directory of the authored component's project.
 
 You'll need to author some files to assist the hosting process by the native app: `YourNativeApp.exe.manifest` and `WinRT.Host.runtimeconfig.json`. 
-
 If your app is packaged with MSIX, then you don't need to include the manifest file, otherwise you need to include your activatable class registrations in the manifest file.
 
-To do this, **in Visual Studio**, right click on the project node on the "Solution Explorer" window, click "Add", then "New Item". Search for the "Text File" template and name your file "YourNativeApp.exe.manifest".
-Repeat this for the "WinRT.Host.runtimeconfig.json" file. 
+To do add these files, **in Visual Studio**, right click on the project node on the "Solution Explorer" window, click "Add", then "New Item". 
+Search for the "Text File" template and name your file `YourNativeApp.exe.manifest`.
+Repeat this for the `WinRT.Host.runtimeconfig.json` file. 
+For each item, right-click on it in the "Solution Explorer" window of Visual Studio; then select "Properties" and change the "Content" property to "Yes" using the drop-down arrow on the right -- this ensures it will be added to the output directory of your solution.
 
-This process adds the nodes `<Manifest Include=... >` and `<None Include=... >` to your native app's project file -- **you need to update these to have `<DeploymentContent>true</DeploymentContent>` for them to be placed in the output directory with your executable**.  
-
-You should read the [hosting docs](https://github.com/microsoft/CsWinRT/blob/master/docs/hosting.md) as well, for more information on these files.
+We have some [hosting docs](https://github.com/microsoft/CsWinRT/blob/master/docs/hosting.md) as well, that provide more information on these files.
 
 In summary, here is the fragment of additions made to the native app's project file:
 ``` vcxproj
-<ItemGroup>
-    <!-- the runtimeconfig.json -->
-    <None Include="WinRT.Host.runtimeconfig.json">
-      <DeploymentContent>true</DeploymentContent>
-    </None>
-    <!-- the manifest -->
-    <Manifest Include="YourNativeApp.exe.manifest">
-      <DeploymentContent>true</DeploymentContent>
-    </Manifest>
-</ItemGroup> 
+<!-- Note: this property group is only required if you are using a project reference, 
+           and is a part of the preview while we work on proper support -->
+<PropertyGroup>
+  <TargetFrameworkVersion>net5.0</TargetFrameworkVersion>
+  <TargetFramework>native</TargetFramework>
+  <TargetRuntime>Native</TargetRuntime>
+</PropertyGroup>
 ```
 
+### C++/WinRT

```

Comment:
Maybe make the header C++ consumption.  there is technically other native scenarios that the component can be consumed in.